x-pack/plugins/aiops/public/components/change_point_detection/metric_field_selector.tsx

```diff
@@ -29,28 +39,32 @@ export const MetricFieldSelector: FC<MetricFieldSelectorProps> = React.memo(
       (selectedOptions: EuiComboBoxOptionOption[]) => {
         const option = selectedOptions[0];
         if (typeof option !== 'undefined') {
-          onChange(option.label);
+          onChange(option.value as string);
         }
+        closeFlyout();
       },
-      [onChange]
+      [onChange, closeFlyout]
     );
 
     return (
-      <EuiFormRow>
-        <EuiComboBox
-          compressed
-          prepend={i18n.translate('xpack.aiops.changePointDetection.selectMetricFieldLabel', {
-            defaultMessage: 'Metric field',
-          })}
-          singleSelection={{ asPlainText: true }}
-          options={options}
-          selectedOptions={selection}
-          onChange={onChangeCallback}
-          isClearable={false}
-          data-test-subj="aiopsChangePointMetricField"
-          onClick={(e) => e.stopPropagation()}
-        />
-      </EuiFormRow>
+      <>
+        <EuiFormRow>
+          <EuiComboBox
```

Comment:
Can you fix the width of this control to a larger value than the current default? With the addition of the icon you don't see much of long field names? Also it changes in width depending on what is selected.

<img width="254" alt="image" src="https://user-images.githubusercontent.com/7405507/233323688-ffbbea30-3bf5-450b-9aa3-08c1d2f54729.png">

Reply:
Updated in https://github.com/elastic/kibana/pull/155177/commits/ae1b9d4f5ae131cc12cb1b008f354d1a1dbb9572